packages/tests/tests/filter/ping.node.spec.ts

```diff
@@ -24,11 +24,13 @@ const runTests = (strictCheckNodes: boolean): void => {
     this.timeout(10000);
     let waku: LightNode;
     let serviceNodes: ServiceNodesFleet;
-    let subscription: IFilterSubscription;
+    let subscription: ISubscriptionSDK;
 
     beforeEachCustom(this, async () => {
       [serviceNodes, waku] = await runMultipleNodes(this.ctx, TestShardInfo);
-      subscription = await waku.filter.createSubscription(TestShardInfo);
+      const { error, subscription: _subscription } =
+        await waku.filter.createSubscription(TestShardInfo);
+      if (!error) subscription = _subscription;
```

Comment:
if error - let's throw from `beforeEachCustom` so that we see it failed to get subscription

Reply:
done, thanks! also updated in other tests